setup/test_examples/xctest.m

```diff
@@ -0,0 +1,24 @@
+@interface Tests : XCTestCase
```

Comment:
Following the pattern of the other templates, the `#import` is in the prefix header (see `.rb` file changes)